jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -500,53 +502,41 @@ public Connection getConnection(String dbPrefix, InterpreterContext context)
     setUserProperty(dbPrefix, context);
 
     final Properties properties = jdbcUserConfigurations.getPropertyMap(dbPrefix);
-    final String url = properties.getProperty(URL_KEY);
+    String url = properties.getProperty(URL_KEY);
+    String principal = getProperty("zeppelin.jdbc.principal");
```

Comment:
I did not notice this earlier, if I remove these three lines, it starts working again as expected.

Reply:
In short for the cluster that I've configured this 
```jdbc:hive2://phivea-3.phivea.root.hwx.site:2181/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=hiveserver2;retries=5;hiveCreateAsExternalLegacy=true;hive.server2.proxy.user=admin;
``` 
works, but 
```jdbc:hive2://phivea-3.phivea.root.hwx.site:2181/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=hiveserver2;retries=5;hiveCreateAsExternalLegacy=true;principal=zeppelin/phivea-1.phivea.root.hwx.site@ROOT.HWX.SITE;hive.server2.proxy.user=admin;
``` 
fails

Reply:
It is weird it fails with principal specified, do you have the exact error message ? IIUC, the principal is required. It should fails if it is not specified.

Reply:
Yes, it works. Not sure may be something changed in the recent version of hive. Should we make this a configurable?

Reply:
Let me revert this change. Originally I just want to remove the redundant principal in `default.url`, because we already ask user to specify principal in `zeppelin.jdbc.principal`, if hive behaivor changes, it is better not to append principal for user

Reply:
@prabhjyotsingh I have removed this change, please help test again.